met-api/src/met_api/models/widget.py

```diff
@@ -74,3 +75,10 @@ def creat_all_widgets(cls, widgets: list) -> list[Widget]:
         db.session.add_all(new_widgets)
         db.session.commit()
         return new_widgets
+
+    @classmethod
+    def remove_widget(cls, engagement_id, widget_id,) -> DefaultMethodResult:
+        """Remove widget from engagement."""
+        widget = Widget.query.filter_by(id=widget_id, engagement_id=engagement_id).delete()
+        db.session.commit()
+        return DefaultMethodResult(True, 'Widget Removed', widget)
```

Comment:
@saravanpa-aot prefers not to use DefaultMethodResult anymore. We could just return widget_id

Reply:
just return widget itself

Reply:
perfect